### ISSUE TITLE
Error: openssl/rsa.h: No such file found

### ISSUE THREAD

**Issue by rajeevmen0n** [imtypist/BlockSense#1]
Error while looking for OpenSSL libraries. 
The issue can be reproduced in Windows by running 

`.\node_modules\.bin\electron-rebuild.cmd`

The following error is obtained
```
C:\Users\username\Documents\Work\DC\BlockSense [master ≡ +1 ~0 -0 !]
λ  .\node_modules\.bin\electron-rebuild.cmd
× Rebuild Failed

An unhandled error occurred inside electron-rebuild
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@11.10.1 | win32 | x64
gyp info spawn C:\Program Files\Python27\python.EXE
gyp info spawn args [ 'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp\\gyp\\gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'msvs',
gyp info spawn args   '-G',
gyp info spawn args   'msvs_version=2015',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\ursa-optional\\build\\config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp\\addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\username\\.electron-gyp\\iojs-3.1.4\\common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=C:\\Users\\username\\.electron-gyp\\iojs-3.1.4',
gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp',
gyp info spawn args   '-Dnode_lib_file=C:\\Users\\username\\.electron-gyp\\iojs-3.1.4\\<(target_arch)\\iojs.lib',
gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\ursa-optional',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\ursa-optional\\build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info spawn C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe
gyp info spawn args [ 'build/binding.sln',
gyp info spawn args   '/clp:Verbosity=minimal',
gyp info spawn args   '/nologo',
gyp info spawn args   '/p:Configuration=Release;Platform=x64' ]
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  ursaNative.cc
  win_delay_load_hook.cc
c:\users\username\documents\work\dc\blocksense\node_modules\ursa-optional\src\ursanative.h(13): fatal error C1083: Cannot open include file: 'openssl/rsa.h': No such file or directory (compiling source file ..\src\ursaNative.cc) [C:\Users\username\Documents\Work\DC\BlockSense\node_modules\ursa-optional\build\ursaNative.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\node-gyp\lib\build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:197:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:254:12)
gyp ERR! System Windows_NT 10.0.17763
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--target=3.1.4" "--arch=x64" "--dist-url=https://atom.io/download/electron" "--build-from-source"
gyp ERR! cwd C:\Users\username\Documents\Work\DC\BlockSense\node_modules\ursa-optional
gyp ERR! node -v v11.10.1
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok

Failed with exit code: 1

Error: gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@11.10.1 | win32 | x64
gyp info spawn C:\Program Files\Python27\python.EXE
gyp info spawn args [ 'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp\\gyp\\gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'msvs',
gyp info spawn args   '-G',
gyp info spawn args   'msvs_version=2015',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\ursa-optional\\build\\config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp\\addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\username\\.electron-gyp\\iojs-3.1.4\\common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=C:\\Users\\username\\.electron-gyp\\iojs-3.1.4',
gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp',
gyp info spawn args   '-Dnode_lib_file=C:\\Users\\username\\.electron-gyp\\iojs-3.1.4\\<(target_arch)\\iojs.lib',
gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\ursa-optional',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\ursa-optional\\build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info spawn C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe
gyp info spawn args [ 'build/binding.sln',
gyp info spawn args   '/clp:Verbosity=minimal',
gyp info spawn args   '/nologo',
gyp info spawn args   '/p:Configuration=Release;Platform=x64' ]
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  ursaNative.cc
  win_delay_load_hook.cc
c:\users\username\documents\work\dc\blocksense\node_modules\ursa-optional\src\ursanative.h(13): fatal error C1083: Cannot open include file: 'openssl/rsa.h': No such file or directory (compiling source file ..\src\ursaNative.cc) [C:\Users\username\Documents\Work\DC\BlockSense\node_modules\ursa-optional\build\ursaNative.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\node-gyp\lib\build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:197:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:254:12)
gyp ERR! System Windows_NT 10.0.17763
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\username\\Documents\\Work\\DC\\BlockSense\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--target=3.1.4" "--arch=x64" "--dist-url=https://atom.io/download/electron" "--build-from-source"
gyp ERR! cwd C:\Users\username\Documents\Work\DC\BlockSense\node_modules\ursa-optional
gyp ERR! node -v v11.10.1
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok

Failed with exit code: 1
    at SafeSubscriber._error (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\spawn-rx\lib\src\index.js:267:84)
    at SafeSubscriber.__tryOrUnsub (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:207:16)
    at SafeSubscriber.error (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:158:26)
    at Subscriber._error (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:92:26)
    at Subscriber.error (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:72:18)
    at MapSubscriber.Subscriber._error (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:92:26)
    at MapSubscriber.Subscriber.error (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:72:18)
    at SafeSubscriber._next (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\spawn-rx\lib\src\index.js:242:65)
    at SafeSubscriber.__tryOrUnsub (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:207:16)
    at SafeSubscriber.next (C:\Users\username\Documents\Work\DC\BlockSense\node_modules\rxjs\internal\Subscriber.js:145:22)
```

**Comment by rajeevmen0n** [comment]
Fixed by PR #2

**Comment by imtypist** [comment]
Thanks for your PR!
